data/json/items/tool/metalworking.json

```diff
@@ -3,35 +3,65 @@
     "id": "anvil",
     "type": "TOOL",
     "name": { "str": "anvil" },
-    "description": "An enormously heavy block of oddly-shaped steel with a chisel-like projection set into the corner.  It's used in most metalworking fabrication recipes.",
-    "weight": "54000 g",
-    "volume": "24 L",
+    "description": "A heavy block of oddly-shaped steel complete with a pritchel hole and hardy hole.  Once installed it can be used in most metalworking fabrication recipes.",
```

Comment:
```suggestion
    "description": "A heavy block of oddly-shaped steel complete with a pritchel hole and hardy hole.  Once installed, it can be used in most metalworking fabrication recipes.",
```